Jenkinsfile_CNP

```diff
@@ -8,10 +8,34 @@ def component = "consumer"
 
 def channel = '#rpe-build-notices'
 
+List<LinkedHashMap<String, Object>> secrets = [
```

Comment:
Could we please test this on CNP branch first before we merge this to master as it will go all the way through prod?

Reply:
Yes, I'll run it from cnp branch before merging. I'll also create keys in Azure Vault.